### PR TITLE
feat: Configure new default page template - EXO-61142 - Meeds/io-MIPs#37

### PR DESCRIPTION
**Pull request by sofyenne**

### CODE REVIEW

File: web/portal/src/main/webapp/WEB-INF/conf/portal/template/pages/normal/page.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+    Lesser General Public License for more details.
+    
+    You should have received a copy of the GNU Lesser General Public
+    License along with this software; if not, write to the Free
+    Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+    02110-1301 USA, or see the FSF site: http://www.fsf.org.
+
+-->
```
Comment (azayati) [inline web/portal/src/main/webapp/WEB-INF/conf/portal/template/pages/normal/page.xml]: it is not the good header

File: web/portal/src/main/webapp/WEB-INF/conf/portal/page-template-configuration.xml [hunk 2]
```diff
@@ -38,6 +38,32 @@
...
+                      </field>
+                      <field name="icon">
+                        <string>NormalLayout</string>
+                      </field>
+                    </object>
+                  </value>
+                  <value>
+                    <object type="org.exoplatform.webui.core.model.SelectItemOption">
+                      <field name="label">
+                        <string>analyticsPage.AnalyticsLayout</string>
```
Comment (azayati) [inline web/portal/src/main/webapp/WEB-INF/conf/portal/page-template-configuration.xml]: this should be under analytics codebase

File: web/portal/src/main/resources/locale/portal/webui_en.properties [hunk 3]
```diff
@@ -649,16 +649,7 @@ UIPageTemplateOptions.UIDropDown.label.tabsPageConfigs=Tabs
 ##################################################################
 UIWizardPageSelectLayoutForm.tab.label.UIPageTemplateOptions=Page template options
 UIWizardPageSelectLayoutForm.label.normalPage.EmptyLayout=Blank Page
-UIWizardPageSelectLayoutForm.label.normalPage.CurrentLayout=Current Layout
```
Comment (azayati) [inline web/portal/src/main/resources/locale/portal/webui_en.properties:652]: - verify if this label is needed
- we should clean the other properties

**Review by azayati — CHANGES_REQUESTED** [review]

File: web/portal/src/main/webapp/WEB-INF/conf/portal/template/pages/normal/page.xml (hunk 1 above)
Comment (sofyenne) [reply web/portal/src/main/webapp/WEB-INF/conf/portal/template/pages/normal/page.xml]: updated !

File: web/portal/src/main/webapp/WEB-INF/conf/portal/page-template-configuration.xml (hunk 2 above)
Comment (sofyenne) [reply web/portal/src/main/webapp/WEB-INF/conf/portal/page-template-configuration.xml]: done !

File: web/portal/src/main/resources/locale/portal/webui_en.properties (hunk 3 above)
Comment (sofyenne) [reply web/portal/src/main/resources/locale/portal/webui_en.properties:652]: cleaned properties !

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**